docs/reference/glossary.md

```diff
@@ -181,22 +187,62 @@ labeled using [tags](#tag).
 Here is an example of the shared [nginx repository](https://hub.docker.com/_/nginx/)
 and its [tags](https://hub.docker.com/r/library/nginx/tags/)
 
-## Swarm
 
-[Swarm](https://github.com/docker/swarm) is a native clustering tool for Docker.
-Swarm pools together several Docker hosts and exposes them as a single virtual
-Docker host. It serves the standard Docker API, so any tool that already works
-with Docker can now transparently scale up to multiple hosts.
+## service
+
+A [service](../swarm/how-swarm-mode-works/services.md) is the definition of how
+you want to run your application containers in a swarm. At the most basic level
+a service  defines which container image to run in the swarm and which commands
+to run in the container. For orchestration purposes, the service defines the
+"desired state", meaning how many containers to run as tasks and where to deploy
```

Comment:
"where to deploy" -> possibly "constraints for deployment" ? (i.e., a service does not allow you to specify _where_ to deploy)